content/docs/next-release/operator.md

```diff
@@ -881,6 +881,25 @@ kubectl delete jaeger simplest
 Deleting the instance will not remove the data from any permanent storage used with this instance. Data from in-memory instances, however, will be lost.
 {{< /info >}}
 
+# Tracing the operator
+
+The Jaeger Operator is able to emit spans for its own operations. By default, the Jaeger Operator ships with an `operator.yaml` that is already configured to send spans to a Jaeger instance named `jaeger`, located in the same namespace as the operator. To use a different instance, change the `operator.yaml` and adjust the flag `reporter.grpc.host-port` to use the appropriate host.
+
+Note that the Jaeger instance has to be manually provisioned and can be done after the Jaeger Operator has been initialized. The Jaeger Agent will keep the spans in the internal buffer until a connection to the instance is made. The following Jaeger CR can be used to provision a Jaeger instance suitable for non-production purposes:
```

Comment:
(passive voice)

Note that you must manually provision the Jaeger instance.  You can do this after the Jaeger Operator has been initialized.  The Jaeger Agent will keep the Operator spans in the internal buffer until it makes a connection to the Jaeger instance.     

[Question - what happens to the spans if that connection is never made?  Are those spans garbage collected?  Or might the buffer overflow at some point?]

Reply:
It uses a circular buffer: older spans are overridden by new spans once if they aren't removed by the "sender" process. No component should ever overflow in any way.